src/jit/jitconfigvalues.h

```diff
@@ -0,0 +1,193 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#if !defined(CONFIG_INTEGER) || !defined(CONFIG_STRING) || !defined(CONFIG_METHODSET)
+#error CONFIG_INTEGER, CONFIG_STRING, and CONFIG_METHODSET must be defined before including this file.
+#endif // !defined(CONFIG_INTEGER) || !defined(CONFIG_STRING) || !defined(CONFIG_METHODSET)
+
+#if defined(DEBUG)
+CONFIG_INTEGER(AltJitLimit, W("AltJitLimit"), 0) // Max number of functions to use altjit for (decimal)
+CONFIG_INTEGER(AltJitSkipOnAssert, W("AltJitSkipOnAssert"), 0) // If AltJit hits an assert, fall back to the fallback JIT. Useful in conjunction with COMPLUS_ContinueOnAssert=1
```

Comment:
In clrconfigvalues.h we used INTERNAL_ or EXTERNAL_ prefix to config values.  As per my understanding the ones with INTERNAL_ prefix are not meant for internal Microsoft development and not supposed to be documented or conveyed external to Microsoft.    I am not sure whether EE/VM does any enforcement of visibility of these config values depending on the prefix.  At least i was told that there is no such enforcement by EE/VM right now, but we could in future if we want.  But nevertheless, those prefixes served as documentation for us.  Now with this change we are losing that info altogether.

How about capturing that info in this list as an additional field, which is ignored for now but will serve as documentation and if required to enforce in future.